pkg/commands/acl/describe.go

```diff
@@ -64,7 +64,7 @@ func (c *DescribeCommand) Exec(_ io.Reader, out io.Writer) error {
 	}
 
 	serviceID, serviceVersion, err := argparser.ServiceDetails(argparser.ServiceDetailsOpts{
-		AllowActiveLocked:  true,
+		DisallowStates:     argparser.DisallowStateActive | argparser.DisallowStateLocked,
```

Comment:
I'm possibly a little confused because we are flipping both the verb and some of the logic. `AllowActiveLocked: true` seems like it should correspond with `DisallowStateNone`, to allow both and disallow none. Is that an incorrect reading?

Reply:
You are not confused, that was a last-minute change before opening the PR for review and I did it very poorly. Will fix!

Reply:
This PR is not ready for review yet, but this area has been completely redesigned and the logic is now... logical.